src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,68 @@
+package storage;
+
+import task.*;
```

Comment:
Try to avoid [wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) whenever possible!